pyproject.toml

```diff
@@ -24,6 +24,7 @@ auth0-python = "^4.4.0"
 algokit-utils = "v2.1.0"
 multiformats = "^0.2.1"
 aiohttp = "3.9.1"
+yaspin = "^3.0.1"
```

Comment:
Is there any web3 storage packages we need to remove?

Reply:
There doesn't seem to be anything to remove.